posthog/api/query.py

```diff
@@ -148,6 +149,9 @@ def create(self, request: Request, *args, **kwargs) -> Response:
                 query_id=client_query_id,
                 user=request.user,  # type: ignore[arg-type]
                 is_query_service=(get_query_tag_value("access_method") == "personal_api_key"),
+                limit_context=LimitContext.APP_INSIGHT_QUERY
```

Comment:
Why can't we use `LimitContext.QUERY` here?

Reply:
`APP_INSIGHT_QUERY` has 600 second timeout instead of 60 seconds for `QUERY`